### DISCUSSION TITLE
"This user account is blocked."

### DISCUSSION THREAD

**Discussion by aju-alen** [vercel/community#3896] [Account Issue] [answered]
When I try to login, I get an error(In title). I tried revoking the access from GitHub and tried to signup again but still fails

**Comment by udaybagda** [comment]
same problem "This user account is blocked."

**Comment by AC90** [comment]
ditto - same problem when trying to connect GitHub account for 1st time.

**Comment by leerob** [comment, marked as answer]
Sorry you're having trouble logging in to your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by udaybagda** [reply]
Still the same problem occurs.

**Comment by masumali23020** [comment]
same problem
